src/ngx_http_lua_log_ringbuf.c

```diff
@@ -12,6 +12,7 @@
 typedef struct {
     unsigned    len:28;     /* :24 is big enough if the max log size is 4k */
     unsigned    log_level:4;
+    double      time;
```

Comment:
@membphis Seems like you forget about field ordering rule again? We should put larger field before smaller field to avoid padding holes.